### PR TITLE
Refine the trend formula in grdtrend using LaTex

### PR DESCRIPTION
**Pull request by core-man**
**Description of proposed changes**

Refine the formula for the trend surface in [grdtrend](https://docs.generic-mapping-tools.org/latest/grdtrend.html) (Originally posted in https://github.com/gmt-china/GMT_docs/issues/238). See the comparisons below:

Current formula:

old

New style:
new

Fixes #

**Reminders**

- [ ] Make sure that your code follows our style. Use the other functions/files as a basis.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Describe changes to function behavior and arguments in a comment below the function declaration.
- [ ] If adding new functionality, add a detailed description to the documentation and/or an example.

### CODE REVIEW

**Comment by PaulWessel** [comment]
Looks good, please add one or more of us as reviewers so we can accept it.

**Comment by core-man** [comment]
> Looks good, please add one or more of us as reviewers so we can accept it.

It seems that I cannot request reviewers in this PR.

PR

**Comment by PaulWessel** [comment]
Hi @seisman, should not posters be able to request a review?

**Comment by seisman** [comment]
> Hi @seisman, should not posters be able to request a review?

Yes, they don't have the permission to request a review or add a label, unless they are given more permissions (e.g., triage permission).

BTW, you can still approve the PR even though no one requests your review. You just need to click the "File Changed" tab, then "Review Changes".

File: doc/rst/source/grdtrend.rst [hunk 1]
```diff
@@ -27,8 +27,9 @@ Description
 to these data by [optionally weighted] least-squares. The trend surface
 is defined by:
 
-   m1 + m2\*x + m3\*y + m4\*x\*y + m5\*x\*x + m6\*y\*y + m7\*x\*x\*x +
-   m8\*x\*x\*y + m9\*x\*y\*y + m10\*y\*y\*y.
+.. math::
+    m_1 + m_2x + m_3y + m_4xy + m_5x^2 + m_6y^2 + m_7x^3 +
+    m_8x^2y + m_9xy^2 + m_{10}y^3.
```
Comment (seisman) [inline doc/rst/source/grdtrend.rst:32]: No need to write the equation in separate lines.
```suggestion
.. math::
    m_1 + m_2x + m_3y + m_4xy + m_5x^2 + m_6y^2 + m_7x^3 + m_8x^2y + m_9xy^2 + m_{10}y^3.
```

**Review by seisman — APPROVED** [review]

**Pull request merged**